### PR TITLE
Format last update date with 'll'

### LINKED ISSUE #42: Change format date of update at on homepage to 'll'

### PR DESCRIPTION
**Pull request by l-vincent-l**
fixes #42

### CODE REVIEW

File: lib/public/components/PublicFeedsViewer.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react'
 import moment from 'moment'
+import 'moment/locale/fr'
```
Comment (bonjourmauko) [inline lib/public/components/PublicFeedsViewer.js:3]: No need, just do:

```javascript
import moment from 'moment/locale/fr';
```

File: lib/public/components/PublicFeedsViewer.js [hunk 2]
```diff
@@ -79,7 +80,7 @@ class PublicFeedsTable extends React.Component {
   }
 
   dateFormat (cell, row) {
-    return cell ? moment(cell).format('MMMM Do YYYY, h:mm a') : ''
+    return cell ? moment(cell).locale('fr').format('ll') : ''
```
Comment (bonjourmauko) [inline lib/public/components/PublicFeedsViewer.js:69]: Same here:

```javascript
return cell ? moment(cell).format('ll') : ''
```

**Review by bonjourmauko — DISMISSED** [review]

**Comment by maukoquiroga** [comment]
Actually no, we need both.

**Comment by bonjourmauko** [comment]
Rebased

**Review by bonjourmauko — DISMISSED** [review]

**Review by bonjourmauko — APPROVED** [review]

**Pull request merged**